mechanicalsoup/form.py

```diff
@@ -277,6 +277,24 @@ def set(self, name, value, force=False):
             return
         raise LinkNotFoundError("No valid element named " + name)
 
+    def __getitem__(self, name):
+        """Forwards arguments to :func:`~Form.get` to get value for a given form field
+        """
+        return self.get(name)
+
+    def get(self, name):
```

Comment:
This lacks a docstring, which is especially frustrating since `__getitem__` explicitly points to this method.

Reply:
Any tips how should i proceed with all these suggestions? git push --force or additional commit?

Reply:
In the end, we'll want a clean history (so given the scope of the PR, most likely only one commit). Sometimes it makes sense to start with additional commits during the review, but `push --force` is probably best here.